From ca7e4b425c0acb6cc32313fcff652dea65718523 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Tue, 19 Jul 2022 17:54:35 +0800
Subject: [PATCH 246/274] arm64: dts: imx93-11x11-evk: set m33 startup delay

set m33 startup to 500ms to make sure when start finish, m33 has
done all the vring initialization

Signed-off-by: Peng Fan <peng.fan@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts b/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
index f599194a7..d77ef92aa 100644
--- a/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-11x11-evk.dts
@@ -68,6 +68,7 @@ &mu1 1 1
 			  &mu1 3 1>;
 		memory-region = <&vdevbuffer>, <&vdev0vring0>, <&vdev0vring1>,
 				<&vdev1vring0>, <&vdev1vring1>, <&rsc_table>;
+		fsl,startup-delay-ms = <500>;
 	};
 
 	reg_can2_stby: regulator-can2-stby {
-- 
2.25.1

